lib/backend/etcdv3/etcdv3.go

```diff
@@ -133,6 +135,14 @@ func (c *etcdV3Client) Create(ctx context.Context, d *model.KVPair) (*model.KVPa
 	logCxt := log.WithFields(log.Fields{"model-etcdKey": d.Key, "value": d.Value, "ttl": d.TTL, "rev": d.Revision})
 	logCxt.Debug("Processing Create request")
 
+	// Can't create the static allow-all profile.
```

Comment:
I think this sort of validation probably belongs up one level in the clientv3 package rather than in the individual backends.